cmd/contour/serve.go

```diff
@@ -1087,6 +1088,18 @@ func (s *Server) setupGatewayAPI(contourConfiguration contour_api_v1alpha1.Conto
 			}
 		}
 
+		// Create and register the TCPRoute controller with the manager.
+		if _, enabled := features["backendtlspolicy"]; enabled {
+			// Inform on ConfigMap if BackendTLSPolicy is enabled
+			if err := s.informOnResource(&corev1.ConfigMap{}, eventHandler); err != nil {
+				s.log.WithError(err).WithField("resource", "configmaps").Fatal("failed to create informer")
+			}
+
+			if err := controller.RegisterBackendTLSPolicyController(s.log.WithField("context", "backendtlspolicy-controller"), mgr, eventHandler); err != nil {
```

Comment:
`internal/k8s/helpers.go#IsObjectEqual` should be updated to include `BackendTLSPolicy`, it should be able to be handled in the same case as the rest of the Gateway API objects.

Reply:
Done.